test/parallel/test-fs-make-callback.js

```diff
@@ -3,37 +3,34 @@ const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 const cbTypeError = /^TypeError: "callback" argument must be a function$/;
+const callbackThrowValues = [null, true, false, 0, 1, 'foo', /foo/, [], {}];
 
-function test(cb) {
+const tmpDir = '/tmp';
```

Comment:
Should this be `os.tmpdir()`? Alternatively, there is `common.tmpDir` for use in tests.